location-service/src/main/java/com/movie/recommender/location/config/LocationServiceConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.movie.recommender.location.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+public class LocationServiceConfig {
+
+    @Bean
+    public RestTemplate restTemplate() {
```

Comment:
replace `RestTemplate` with `HttpClient` from `com.movie.recommender.lib.http` (merge main into your branch to have this lib)
`LocationServiceConfig` should be deleted when you switch to `HttpClient`